src/db/spanner/support.rs

```diff
@@ -215,26 +215,21 @@ impl SyncResultSet {
         }
     }
 
-    #[cfg(feature = "google_grpc")]
-    pub fn affected_rows(self: &SyncResultSet) -> Result<i64> {
-        let stats = self
-            .stats()
-            .ok_or_else(|| DbError::internal("Expected result_set stats"))?;
-        Ok(stats.get_row_count_exact())
+    pub fn all_or_none(&mut self) -> Option<Vec<ListValue>> {
```

Comment:
We can revisit this later but I'm not sure why this method is necessary w/ SyncResult supporting Iteration. That let's you do something like `let result: Vec<_> = sql(..)?.execute(..)?.collect();`, giving you a `Vec<Vec<Value>>` vs `Vec<ListValue>`

Reply:
Ok, awesome. I did not know that.